data/brands/amenity/vending_machine.json

```diff
@@ -383,16 +380,6 @@
         "vending": "excrement_bags"
       }
     },
-    {
-      "displayName": "Schwalbe",
```

Comment:
Isn't this one real?  They make bicycle tubes..

Reply:
Okay cool! I'll look through the missing tags now! Thank you :)!